.gitignore

```diff
@@ -31,8 +194,7 @@ assets/private.yml
 assets/garbage
 **.log
 log/**
-
-XAgentWeb/src/api/backend.ts
+config.*
```

Comment:
Why do we need `config.*` ?

Reply:
The API keys are configured in `config.yml`, so it must be ignored. The better specification is to split default_config, user config and secrets, which should be in the task list.

Reply:
I get it. But we already ignored those file. I thought this wildcard is going to ignore like `config.toml, config.temp` etc?
```yml
assets/config.yml
assets/gpt4_config.yml
config.yml
```